awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -67,8 +76,28 @@ class WorkflowJobTemplate extends Component {
         );
         data.summary_fields.webhook_credential.kind = name;
       }
+      const [notifAdminRes, auditorRes, adminRes] = await Promise.all([
+        OrganizationsAPI.read({
+          page_size: 1,
+          role_level: 'notification_admin_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'auditor_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'admin_role',
+        }),
+      ]);
```

Comment:
https://github.com/ansible/awx/pull/6185/files#diff-b91ff4e4db2d2c5d939ecbb4b9fc8353R58-R92

This method is now too big and we should break it up into smaller function calls.